src/breadcrumb.scss

```diff
@@ -54,6 +54,30 @@ $fd-breadcrumb-border-top-radius: 0.125rem !default;
 
     display: inline-block;
     transition: all $fd-animation-speed ease-in;
+
+    &:focus {
+      .#{$block}__dropdown-icon {
+        color: var(--sapLink_Active_Color, #0a6ed1);
+      }
+    }
+
+    &:active {
+      .#{$block}__dropdown-icon {
+        color: var(--sapLink_Active_Color, #0a6ed1);
+      }
+
+      &:hover {
+        .#{$block}__dropdown-icon {
+          color: var(--sapLink_Active_Color, #0a6ed1);
+        }
+      }
+    }
+
+    &:hover {
+      .#{$block}__dropdown-icon {
+        color: var(--sapLink_Hover_Color, #0854a0);
```

Comment:
Why are you adding the colors this way? 
Should be just color: `var(--sapLink_Hover_Color);` or `color: var(--sapLink_Active_Color)

Reply:
and since `var(--sapLink_Active_Color)` is repeated 3 times maybe put it in a variable.

Reply:
I noticed we have fallbacks in some components but ill change it :)

Reply:
That's not the proper way to add a fallback in general. You will need:
```
color:  #0854a0;
color: var(--sapLink_Hover_Color, #0854a0);
```

Reply:
Also, I searched the project and couldn't find such case :)
There's one usage in `custom.scss` file which is for the documentation. I think you are talking about fundamental-ngx, but this is another case.